waterdata/tests/test_views.py

```diff
@@ -1,15 +1,21 @@
+"""
+Unit tests for the main WDFN views.
+"""
 
 from unittest import TestCase
 
 from .. import app
 from ..views import __version__
 
+
 class TestHomeView(TestCase):
+    """Basic tests validating existence of pages."""
```

Comment:
I'm not sure how necessary doc string comments are in tests. Test case classes should be tied to a particularly module or class and the methods themselves should be small and the function name should be descriptive

Reply:
I don't disagree, but we could strive to have zero linting errors... it appears to not actually be possible (yet) to have different .pylintrc files in subdirectories: https://github.com/PyCQA/pylint/issues/618

Reply:
We could try adding no-docstring-rgs: '^test\w|^Test\w' (I think that would work. Not the best at regex)

Reply:
Ah! I didn't know we could do that in pylint. I'll push another commit.